### ISSUE TITLE
`require': cannot load such file -- curses (LoadError)

### ISSUE THREAD

**Issue by dentarg** [etsy/mctop#21]
I have problems running mctop with Ruby 2.1.2p95 (on OS X):

``` sh
$ ./bin/mctop --help
/Users/dentarg/.rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- curses (LoadError)
    from /Users/dentarg/.rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/dentarg/src/mctop/lib/ui.rb:1:in `<top (required)>'
    from /Users/dentarg/.rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/dentarg/.rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from ./bin/mctop:12:in `<main>'
```

Probably due to the fact that curses was removed from standard library in Ruby 2.1.0: https://bugs.ruby-lang.org/issues/8584

**Comment by Bsmalhi** [comment]
Hi Guys,
I would like a little help here I am getting this error when doing ruby curses.rb
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- curses (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from curses.rb:3:in `<main>'
Could you please point me to what needs to be fixed I would really appreciate it thanks :)

**Comment by keyurdg** [comment]
@Bsmalhi mctop is no longer a supported project. If the fix mentioned above (48b66d057466e6bd5626f1c748dfbee61bca954e) does not work, then consider using https://github.com/tumblr/memkeys
